cms/urls.py

```diff
@@ -347,6 +347,6 @@
 # Custom settings are provided in SPECTACULAR_SETTINGS in cms/envs/common.py.
```

Comment:
```suggestion
# Custom settings are provided in SPECTACULAR_SETTINGS as environment variables
```
Also, I know that this is not your doing, but the fact that SPECTACULAR_SETTINGS have to be defined as env variables and can't be overwritten per API is kind of frustrating no?

Reply:
It is. I'm hoping there is a solution somewhere in spectacular for it, so that we can keep our separate swagger ui for the cms api later on after we switch to spectacular completely.